tests/std/tests/VSO_0226079_mutex/test.cpp

```diff
@@ -148,12 +148,9 @@ class other_mutex_thread {
 
 template <typename Func>
 nanoseconds time_execution(Func&& f) {
-    // system_clock currently powers mutex waits, so we're using system_clock
-    // rather than high_resolution_clock here for consistency. This should be
-    // fixed with VSO-133414, VSO-166543, VSO-189735.

```

Comment:
No change requested: For the record, these were

* VSO-133414 `<condition_variable>`: `wait_until()` is implemented with relative time
* VSO-166543 `<condition_variable>`: `wait_until` and `sleep_until` functions don't behave correctly for user-defined clocks
* VSO-189735 `<thread>`: `std::this_thread::sleep_for` hangs when system time is moved backwards

which indeed should all be fixed now.